gulpfile.js

```diff
@@ -0,0 +1,33 @@
+var gulp = require('gulp');
+  connect = require('gulp-connect');
+
+gulp.task('default', function() {
+  connect();
+});
+
+gulp.task('connect', function() {
+  connect.server({
+    root: './',
+    port: 1820,
+    livereload: true
+  });
+});
+
+gulp.task('reload', function () {
+  gulp.src('./**')
+    .pipe(connect.reload());
+});
+
+gulp.task('watch', function () {
+  gulp.watch(['./Unit2/*.html'], ['reload']);
+});
+
+gulp.task('my-tests', function() {
+  connect.server({
+    port: 1820
+  });
+
+  connect.serverClose();
```

Comment:
Is this line neccessary? I wouldn't think you would want to close the connect server. You would want to keep it open to view your changes on localhost:1820. I think this will only present a problem when you get deeper into Gulp commands.